From 462db92b2ce6645876f90fd7b6dfc423c965cc6d Mon Sep 17 00:00:00 2001
From: yanghuatao <yanghuatao@xiaomi.com>
Date: Thu, 23 Mar 2023 09:47:17 +0800
Subject: [PATCH 2/3] fix build error: do not include config.h

VELAPLATFO-6159

config.h is generated by confugure, we do not need it

(1)open menuconfig  (2)close math.h:build setup->Customize Header Files->math.h (2) select libmcs:Library Rountines->Select math library->Math Library fram LibmCS (3)build

Signed-off-by: yanghuatao <yanghuatao@xiaomi.com>
Change-Id: I2d6efed47a8addee7624cbb972fb2ca914c1684c
---
 libmcs/libmcs/libm/include/complex.h | 1 -
 libmcs/libmcs/libm/include/math.h    | 1 -
 2 files changed, 2 deletions(-)

diff --git a/libmcs/libmcs/libm/include/complex.h libmcs/libmcs/libm/include/complex.h
index ab30ac979..5a2ba59ac 100644
--- a/libmcs/libmcs/libm/include/complex.h
+++ libmcs/libmcs/libm/include/complex.h
@@ -15,7 +15,6 @@
 extern "C"{
 #endif
 
-#include "config.h"
 #include "internal_config.h"
 
 #ifndef __LIBMCS_WANT_COMPLEX
diff --git a/libmcs/libmcs/libm/include/math.h libmcs/libmcs/libm/include/math.h
index 1bb0e4bab..b0f6427df 100644
--- a/libmcs/libmcs/libm/include/math.h
+++ libmcs/libmcs/libm/include/math.h
@@ -5,7 +5,6 @@
 extern "C"{
 #endif
 
-#include "config.h"
 #include "internal_config.h"
 
 /*
-- 
2.39.1

